### ISSUE TITLE
Sum of trades in price chart uses incorrect price / Summe von Trades im Kursdiagramm nutzt falschen Kurs

### ISSUE THREAD

**Issue by imperatom** [portfolio-performance/portfolio#3854]
**- English version below -** 
Im Kursdiagramm von Wertpapieren werden praktischerweise Trades als senkrechte Linien dargestellt. Als Tooltip erscheint dort z.B. “1000 Stück zu 5,00 Euro”. Dementsprechend sollte in der Zeile darüber hinter dem Summenzeichen 1000 x 5 = 5000 Euro stehen.

Tatsächlich wird aber offenbar nicht mit dem Verkaufskurs, sondern mit dem gespeicherten Tageskurs multipliziert, so dass - insbesondere an schwankungsreichen Tagen - eine deutliche Abweichung zur Realität entsteht. 

Der Bug sollte leicht behebbar sein durch Anpassung der genutzten Variable, ggf. muss allerdings beachtet werden, dass mehrere Trades am gleichen Tag stattfanden.

----
**google translator says:**
In the price chart of securities, trades are conveniently shown as vertical lines. For example, “1000 pieces for 5.00 euros” appears as a tooltip. Accordingly, in the line above, behind the sum symbol, there should be 1000 x 5 = 5000 euros.

In fact, however, it is apparently not multiplied by the sales rate, but by the saved daily rate, so that - especially on days with high fluctuations - there is a clear deviation from reality.

The bug should be easy to fix by adjusting the variable used, although it may be important to note that several trades took place on the same day.

**Comment by Morpheus1w3** [comment]
PC oder Smartphone?

**Comment by imperatom** [comment]
PC - Windows. 0.68.2

**Comment by Sn1kk3r5** [comment]
Hi @imperatom,

ich kann dir nicht ganz folgen. Kannst du mal bitte einen Screenshot einstellen. Das von dir beschrieben verhalten kann ich bei mir nicht nachstellen. Ich sehe im Tooltipp genau den Veraufspreis.

Viele Grüße

**Comment by imperatom** [comment]
Ich bin dem nochmal genauer nachgegangen und habe dabei festgestellt, dass ich den Fehler auf die falsche Ursache geschoben hatte - das tut mir leid! Tatsächlich gibt es zwar die Abweichung, es wird aber offenbar der richtige Kurs genommen. Dann ist mir aufgefallen, dass die Abweichung offenbar nur bestimmte Fälle bertrifft, nämlich Verkäufe, bei denen Steuern gezahlt worden sind. 

Dass die dargestellte "Summe" nicht erwartungsgemäß dem Produkt aus Kaufpreis und Stückzahl entspricht, liegt nur daran, dass dabei noch Steuern abgezogen werden, was sehr irritierend ist, weil es überhaupt keinen Hinweis darauf gibt, dass dabei Steuern berücksichtigt werden würden. Die Sache ist also anders als zunächst von mir vermutet. Ich fände es sehr hilfreich, wenn hinter dem Summenzeichen im Tooltip künftig der Verkaufspreis VOR Steuern angezeigt würde oder es alternativ einen Hinweis auf die abgezogene Steuer gäbe, denn eigentlich erwartet man dort kene Steuerabzüge.

**Comment by Sn1kk3r5** [comment]
Dann kannst du das Bug Label entfernen. 

Tooltips enthalten idR keine Detail Informationen. Dafür gibt es die Detailsicht zum Beispiel unter Trades. 

Viele Grüße

**Comment by imperatom** [comment]
Ja, es wäre daher gut, einfach das Vorsteuerergebnis der Verkäufe dort darzustellen.

Die Bug-Plakette kann von mir aus entfernt werden, ich finde keinen Knopf dafür.

**Comment by Sn1kk3r5** [comment]
> Ja, es wäre daher gut, einfach das Vorsteuerergebnis der Verkäufe dort darzustellen.

Ich verstehe deinen Punkt, das Problem hierbei ist, jeder Nutzer hat seinen eigenen Blick samt Erwartung.
